app/controllers/legend_panes_controller.rb

```diff
@@ -1,12 +1,11 @@
-class MapKeysController < ApplicationController
+class LegendPanesController < ApplicationController
   before_action :authorize_web
   before_action :set_locale
   authorize_resource :class => false
 
   def show
-    expires_in 7.days, :public => true
```

Comment:
I don't think this was particularly because of re-requests from a single session, it was just that it was something that shouldn't be changing much so there was no reason not to cache it. It came in from 817434a2e6f1510bf805e35f0c879b005ebe11dd originally.

Reply:
There are reasons not to cache it, but yes, it can be removed in another PR.